### DISCUSSION TITLE
How to disable the popup menu?

### DISCUSSION THREAD

**Discussion by NooBiToo** [vueComponent/ant-design-vue#6695] [Q&A] [answered]
![Screenshot_1](https://github.com/vueComponent/ant-design-vue/assets/12481334/9a15fc93-81b3-432d-aff9-a877cf3ec6ba)
need disabled this

**Comment by daiwanxing** [comment]
didn't understand your question, need more describe.

**Reply by NooBiToo** [reply]
i use a collapsed menu, but after reloading the page, this popup menu appears on the side
I don't know how to turn it off

**Reply by NooBiToo** [reply]
```
<template>
  <a-menu class="q-sidebar" :open-keys="state.openKeys" mode="inline" v-model:selectedKeys="state.selectedKeys" @click="сlickShowPage">
    <a-sub-menu v-for="item in items" :key="item.path" @titleClick="titleClick($event, item.path)">
      <template #icon>
        <component :is="item.meta.icon" />
      </template>
      <template #title>
        {{ item.name }}
      </template>
      <a-menu-item v-for="subItem in item.children" :key="subItem.path">
        {{ subItem.name }}
      </a-menu-item>
    </a-sub-menu>
  </a-menu>
</template>
```

**Reply by NooBiToo** [reply]
![image](https://github.com/vueComponent/ant-design-vue/assets/12481334/0895c4a8-8be1-4939-b5d1-4a3ce2be41d0)

**Reply by NooBiToo** [reply]
![image](https://github.com/vueComponent/ant-design-vue/assets/12481334/c395119a-90e2-475d-b7a7-12929637127c)

**Reply by daiwanxing** [reply]
could you please share a live demo on codesandbox or other online editor? This would solve the problem better if it were possible

**Reply by NooBiToo** [reply]
![image](https://github.com/vueComponent/ant-design-vue/assets/12481334/87d23356-2aa3-4d98-b53e-4da45d24cd42)
need disabled this
because it appears after page reload

**Reply by daiwanxing** [reply]
It's seems something change the `<a-menu>` mode property，please give me a minal live demo.

**Reply by NooBiToo** [reply]
that's a lot of code =(

**Reply by daiwanxing** [reply]
If the mode attribute of the `<menu>` component is set to `inline`, the submenus will open as pop-ups. Based on the given clues, it seems that your code controls the menu mode when the page is reloaded.

**Reply by daiwanxing** [reply]
But you can't give a minimal reproduction, I can't continue analyzing the problem.

**Comment by NooBiToo** [comment, marked as answer]
:internal-popup-close="true" for <a-sub-menu>
